### PR TITLE
ref(ui) Consolidate onboarding state styles

### PR DESCRIPTION
**Pull request by markstory**
Use the same components for the releases & performance onboarding states. This new component may see more action in the future as onboarding states are revisited.

### CODE REVIEW

**Review by matejminar — APPROVED** [review]
❤️

File: src/sentry/static/sentry/app/components/onboardingPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {Panel} from 'app/components/panels';
+import space from 'app/styles/space';
+
+type Props = React.PropsWithChildren<{
+  className?: string;
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/onboardingPanel.tsx:8]: as classname is optional I would move it under the image prop

File: src/sentry/static/sentry/app/components/onboardingPanel.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {Panel} from 'app/components/panels';
+import space from 'app/styles/space';
+
+type Props = React.PropsWithChildren<{
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/onboardingPanel.tsx:7]: never used it.. nice :)

**Review by priscilawebdev — APPROVED** [review]
LGTM ✨

**Review by dashed — APPROVED** [review]
Looks good to me 👍

**Pull request merged**